session/example_test.go

```diff
@@ -0,0 +1,62 @@
+package session_test
+
+import (
+	apiGoSession "github.com/nspcc-dev/neofs-api-go/v2/session"
+	cid "github.com/nspcc-dev/neofs-sdk-go/container/id"
+	neofscrypto "github.com/nspcc-dev/neofs-sdk-go/crypto"
+	"github.com/nspcc-dev/neofs-sdk-go/session"
+	"github.com/nspcc-dev/neofs-sdk-go/user"
+)
+
+// Both parties agree on a secret (private session key), the possession of which
+// will be authenticated by a trusted person. The principal confirms his trust by
+// signing the public part of the secret (public session key).
+func ExampleContainer_Sign() {
+	// import neofscrypto "github.com/nspcc-dev/neofs-sdk-go/crypto"
+	// import "github.com/nspcc-dev/neofs-sdk-go/user"
+	// import cid "github.com/nspcc-dev/neofs-sdk-go/container/id"
+
+	// you private key/signer, to prove you are you
+	var principalSigner user.Signer
+	// trusted party, who can do action on behalf of you
```

Comment:
Usually from `SessionCreate`. Do we have an example for `SessionCreate`, btw? That'd be more helpful for people.